models/rooms.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(sequelize, DataTypes) {
+    const Rooms = sequelize.define("Rooms", {
```

Comment:
a typical naming convention in sequelize for tables and variables that hold the tables are as follows: in our models code we are "modeling" one item so we use the name Room e.g.:
const Room = sequelize.define("Room",{ Sequelize will automatically make the name of the table Rooms, but now our model reads a little better with "db.Room"

Reply:
If you change this, be sure to change it throughout your entire project